### PR TITLE
Reactivate library

### PR DESCRIPTION
**Pull request by holgerd77**
Various fixes to reactivate the library.

### CODE REVIEW

**Comment by holgerd77** [comment]
@fanatid Are you comfortable with when I change ``es/`` directory to ``lib/`` and ``lib/`` to ``build/``? Would be more in-line with the other ``ethereumjs`` libraries.

**Comment by fanatid** [comment]
I'd prefer `src` instead `es`, but if you use `lib` everywhere -- I'm ok with this.

**Comment by fanatid** [comment]
Please update babel to `babel-presets-env` with node >= 6 (or 4?)

**Comment by holgerd77** [comment]
@fanatid Ok, I think I'm ready now, also updated to the ``env`` preset. Can you do a short review?

File: .babelrc [hunk 1]
```diff
@@ -1,8 +1,7 @@
 {
+  "presets": ["env"],
```
Comment (fanatid) [inline .babelrc]: I think should be:
```json
["env", {
      "targets": {
        "node": "6"
      }
    }]
```

**Comment by fanatid** [comment]
@holgerd77 please add [field engines to package.json](https://docs.npmjs.com/files/package.json#engines), because we do not support node < 6

**Comment by fanatid** [comment]
and can we add `safe-buffer`? other looks good!

**Comment by fanatid** [comment]
I added you and @axic to owners on npm.

File: .babelrc (hunk 1 above)
Comment (holgerd77) [reply .babelrc]: Ah, I didn't really grasped this, babel is actually used to transpile for node?

Then my renaming with the ``lib`` folders really doesn't make sense, I will revert this and change it to your version with naming ``src``.

Comment (fanatid) [reply .babelrc]: babel transpile code for every `npm publish`
+1 for `lib` to `src`
maybe `build` to `lib`?

Comment (holgerd77) [reply .babelrc]: Yes yes, of course also ``build`` back to ``lib``, didn't get the whole concept. :-)

Comment (holgerd77) [reply .babelrc]: Done.

**Comment by holgerd77** [comment]
@fanatid Ok, implemented all your changes, maybe have a last look at it.

**Comment by holgerd77** [comment]
And thanks for adding me on npm, great!

**Comment by fanatid** [comment]
Thanks. Published as 2.1.3

**Pull request merged**